lib/SILGen/SILGenApply.cpp

```diff
@@ -5822,6 +5824,9 @@ RValue SILGenFunction::emitGetAccessor(SILLocation loc, SILDeclRef get,
   CanAnyFunctionType accessType = getter.getSubstFormalType();
 
   CallEmission emission(*this, std::move(getter), std::move(writebackScope));
+  if (implicitActorHopTarget)
+    emission.setImplicitlyAsync(implicitActorHopTarget);
+
```

Comment:
I like this approach of moving the responsibility into the CallEmission. I didn't know it had `setImplicitlyAsync`.